File: Python Excel Examples/CellsApiDemo/row/setRowStyle.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "SetRowStyle_1.xlsx"
sheetName = "Sheet2"
row_index = 3
style = spirecloudexcel.models.Style()
font = spirecloudexcel.models.Font()
font.underline = "Single"
font.size = 8
font.is_italic = True
font.is_bold = True
font.name = "Algerian"
style.font = font
borders = []
topBorder = spirecloudexcel.models.Border("Medium", spirecloudexcel.models.Color(255, 255, 0, 0), "EdgeTop")
rightBorder = spirecloudexcel.models.Border("DashDot", spirecloudexcel.models.Color(255, 0, 255, 0),
                                            "EdgeRight")
borders.append(topBorder)
borders.append(rightBorder)
style.border_collection = borders
style.horizontal_alignment = "Center"
style.background_color = spirecloudexcel.models.Color(255, 0, 255, 0)
storage = ""
folder = "/Cells/Row/"
api.set_row_style(name, sheet_name=sheetName, row_index=row_index, style=style, folder=folder, storage=storage)

File: JavaScript Excel Examples/CellsApiDemo/Row/setRowStyle.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel= require('../../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.CellsApi(apiClient);

    var name = "Rows.xlsx";
    var sheetName = "Sheet1";
    var rowIndex = 3;
    var style = new SpirecloudExcel.Style();
    style.Font = new SpirecloudExcel.Font();
    style.Font.IsBold = true;
    style.Font.IsItalic = true;
    style.Font.Underline = "Single";
    style.Font.Name = "Algerian";
    style.Font.Size = 8;
    style.BackgroundColor = new SpirecloudExcel.Color();
    style.BackgroundColor.A = 255;
    style.BackgroundColor.R = 0;
    style.BackgroundColor.G = 255;
    style.BackgroundColor.B = 0;
    style.HorizontalAlignment = "Center";
    style.BorderCollection = [new SpirecloudExcel.Border(), new SpirecloudExcel.Border()];
    style.BorderCollection[0].LineStyle = "Medium";
    style.BorderCollection[0].Color = new SpirecloudExcel.Color();
    style.BorderCollection[0].Color.A = 255;
    style.BorderCollection[0].Color.R = 255;
    style.BorderCollection[0].Color.G = 0;
    style.BorderCollection[0].Color.B = 0;
    style.BorderCollection[0].BorderType = "EdgeTop";

    style.BorderCollection[1].LineStyle = "DashDot";
    style.BorderCollection[1].Color = new SpirecloudExcel.Color();
    style.BorderCollection[1].Color.A = 255;
    style.BorderCollection[1].Color.R = 0;
    style.BorderCollection[1].Color.G = 255;
    style.BorderCollection[1].Color.B = 0;
    style.BorderCollection[1].BorderType = "EdgeRight";

    var opts = { folder: "input"};
    instance.setRowStyle(name, sheetName, rowIndex, style, opts,
        function (error, data, response) {
            if (error) throw error;
        });
})();